### ISSUE TITLE
Are you ok?

### ISSUE THREAD

**Issue by 68000ft** [sunjon/stylish.nvim#6]
We can't reach you.

Please to the network of the DDT, and, well, if you curently are visiting DDT, then... 

say hi?

But would prefer if you could just pop in and say "hi I'm busy, talk later" than if you were visiting DDT.

Yes, we tried ... all other methods.

**Comment by nyngwang** [comment]
Sad...

**Comment by 68000ft** [comment]
I did reach him.  Tho I haven't heard from him lately.

**Comment by nyngwang** [comment]
When? So (s)he gave up using GitHub?

**Comment by 68000ft** [comment]
No.  Life gets in the way of things very often, and this is one of those times.
